errors.go

```diff
@@ -0,0 +1,11 @@
+package moira
+
+//SenderBrokenContactError means than sender has no way to send message to contact.
+// Maybe receive contact was deleted, blocked or archived.
+type SenderBrokenContactError struct {
+	SenderError error
+}
```

Comment:
I wish to see a `NewSenderBrokenContactError` function here that will be used in senders to create a new errors instead of creating new errors via structure literal

Reply:
fixed